kuma/attachments/jinja2/attachments/includes/attachment_list.html

```diff
@@ -105,6 +105,8 @@
               <a href="{{ document_attachment.document.get_absolute_url() }}">
                 {{ document_attachment.document.title }} ({{ document_attachment.document.locale}})
               </a>{% if not loop.last %}, {% endif %}
+            {% else %}
+              {{ _('n/a') }}
```

Comment:
Yeah, exactly this use case, or for those handful of attachments the script wasn't able to find any documents in question (either because the documents were deleted, or some other data inconsistency). So this is both for historical attachments as well as new attachments in edge cases